piccolo_admin/endpoints.py

```diff
@@ -401,6 +401,7 @@ def __init__(
         translations: t.List[Translation] = None,
         allowed_hosts: t.Sequence[str] = [],
         debug: bool = False,
+        sidebar_links: t.List[t.Tuple[str, str]] = [],
```

Comment:
I think this was my fault, suggesting tuples. But if we just the user pass in a dict instead, we no longer need `get_sidebar_links`.

```
sidebar_links={
    "Top Movies": "/movies/?order=-popularity
```

Reply:
> Am I missing something?

No, you're right - I just meant we didn't need the bit that mapped tuples into dictionaries any more, as they're already dictionaries.